model/godleyTableWindow.cc

```diff
@@ -94,11 +94,19 @@ namespace minsky
           godleyIcon.table.moveRow(idx,0); 
         else if (pos==second)                  // Third button of second row cannot swap initial conditions and second row. For ticket 1064
            godleyIcon.table.moveRow(idx,1); 
+<<<<<<< HEAD
+        else if (pos!=first && pos!=firstAndLast)                  // Third button cannot swap column headings and initial conditions row values. For ticket 1064     
+          godleyIcon.table.moveRow(idx,-1);   
+        break;
+      case 3:
+        if (pos==middle)             // Fourth button on first and second row cannot move initial conditions row. For ticket 1064
+=======
         else if (pos !=first)                  // Third button cannot swap column headings and initial conditions row values. For ticket 1064     
           godleyIcon.table.moveRow(idx,-1);   
         break;
       case 3:
         if (pos==second || pos==middle)             // Fourth button on first and second row cannot move initial conditions row. For ticket 1064
+>>>>>>> e0543ca9043a4b33cdf459b9ce96c719a8bc8450
```

Comment:
Obviously a missing conflict resolution.

Reply:
Thank you, I have removed the conflicting lines, but travis has also picked up ambiguous parentheses in if statements which I've just fixed. I hope the code will build now...